code-of-conduct.md

```diff
@@ -0,0 +1,62 @@
+---
+# You don't need to edit this file, it's empty on purpose.
```

Comment:
Should this comment be removed or updated? It doesn't seem to be true. This file is not empty and has relevant content. Or is the comment referring to the `layout: coc`? In any case this is a bit confusing to me.

Reply:
Ha, good point! Classic copy-pasting from old stuff.

Reply:
This is still there in this file. You removed it only from `index.md`.